File: competitive-events-api/src/main/java/es/jocasolo/competitiveeventsapi/dto/image/ImageDTO.java
package es.jocasolo.competitiveeventsapi.dto.image;

import java.io.Serializable;

import es.jocasolo.competitiveeventsapi.dto.DTO;
import lombok.Getter;
import lombok.Setter;

public class ImageDTO extends DTO implements Serializable {

	private static final long serialVersionUID = 1L;

	@Getter
	@Setter
	private Integer id;
	
	@Getter
	@Setter
	private String storageId;

	@Getter
	@Setter
	private String name;

	@Getter
	@Setter
	private String url;

	@Override
	public String toString() {
		return "ImageDTO [name=" + name + "]";
	}

}

File: competitive-events-app/app/src/main/java/es/jocasolo/competitiveeventsapp/dto/user/ImageDTO.kt
package es.jocasolo.competitiveeventsapp.dto.user

import com.google.gson.annotations.SerializedName

data class ImageDTO(
    @SerializedName("id") var id: String,
    @SerializedName("name") var name: String,
    @SerializedName("url") var url: String
)